ros_tcp_endpoint/server.py

```diff
@@ -48,15 +48,17 @@ def __init__(self, node_name, buffer_size=1024, connections=10, tcp_ip="", tcp_p
         """
         super().__init__(node_name)
 
-        self.declare_parameter("ROS_IP", "127.0.0.1")
+        self.declare_parameter("ROS_IP", "0.0.0.0")
```

Comment:
Should this be the default? It's only valid in docker.

Reply:
I guess it depends on what we think our split is between docker and non-docker use cases.  Since this is always the first fix we suggest when users are having network issues, it felt like it should be our default.